examples/functions/algolia-document-sync/README.md

```diff
@@ -63,11 +63,11 @@ This function is built to be compatible with any of [the official "clean" templa
      event: {
        on: ['publish'],
        filter: "_type == 'post'",
-       projection: '_id',
+       projection: '_id, title, hideFromSearch',
```

Comment:
Needed to update the projection, otherwise your function would not get the data it is expecting. The way it is set currently you'd only get the `_id`.